assets/js/components-noncritical.js

```diff
@@ -38,6 +39,11 @@ const componentConfiguration = [
       optionalListToggle: '.optional-list-toggle',
       rotateArrowClass: 'rotate-arrow',
       shownClass: 'shown',
+      cookieModal: '.ng-cc-modal',
+      cookiePolicyShow: '.ng-cc-js-link-cookie-policy a',
```

Comment:
Is it possible to handle this to put the .ng-cc-js-link-cookie-policy class on link itself and not on wrapper and use the link as this value?

cookiePolicyShow: '.ng-cc-js-link-cookie-policy', // where html is <a href="#" class="ng-cc-js-link-cookie-policy">text</a>